### ISSUE TITLE
Code incorrectly reports E0034 on `fmt`

### ISSUE THREAD

**Issue by Enrico2** [intellij-rust/intellij-rust#5457]
## Environment

* **IntelliJ Rust plugin version:** 0.2.122.3105-201
* **Rust toolchain version:** 1.45.0-nightly (a74d1862d 2020-05-14) x86_64-apple-darwin
* **IDE name and version:** IntelliJ IDEA 2020.1.1 Ultimate Edition (IU-201.7223.91)
* **Operating system:** macOS 10.15.4

## Problem description

The offending line is here: https://github.com/apollographql/apollo-cli/blob/efb25c783a4e0108a34398c85cdebe43f145d6e2/cli/src/errors/error.rs#L48

## Steps to reproduce

Clone the repo, checkout that commit, look at that line. This is the error:

![image](https://user-images.githubusercontent.com/161314/82941802-dac61680-9f4b-11ea-9a24-255aa0fcb446.png)

the type of `self.inner.kind` is `ErrorDetails` which only has 
```rust
#[derive(Error, Debug)]
pub enum ErrorDetails { ... }
```

cargo build passes:
```
cargo build --color=always --all --all-targets
   Compiling graphql-parser v0.3.0 (/Users/ran/dev/apollo/apollo-cli/graphql-parser)
   Compiling apollo-cli v0.0.3 (/Users/ran/dev/apollo/apollo-cli/cli)
    Finished dev [unoptimized + debuginfo] target(s) in 5.23s

Process finished with exit code 0
```
